src/universal/config/myarea-datasets.ts

```diff
@@ -86,20 +87,23 @@ export const DATASETS: DatasetCategories = {
             },
           },
           CHADEMO: {
+            group: 'Connector type',
```

Comment:
handige aan deze aanpak is dat je alleen een group hoeft toe te voegen. Dan wordt er een object gemaakt met keys voor elke groep en daarbinnen de filters die daarbij horen. Heb wel de keuze gemaakt de groepen als eerste te renderen en daarna filters zonder groep. Je zou kunnen stellen dat de volgorde in de datasets config lijdend moet zijn. Dan moet er een aanpassing worden gemaakt.

Reply:
Hoi Oscar, ik heb er naar gekeken en de volgorde in de datasetconfig is inderdaad wel belangrijk. Ik wil toch ook kijken naar een meer userfriendly manier om die Ja / Nee opties vorm te geven. Ik sluit 'm voor nu. Als je terug bent kunnen we een story refinen voor iets meer sophisticated grouping en mooiere Ja / Nee filters.